zipline/algorithm.py

```diff
@@ -608,6 +608,11 @@ def _calculate_universe(self):
         # our universe is all the assets passed into `run`.
         return self._assets_from_source
 
+    def _compute_pipelines(self):
+        for name, p in self._pipelines.items():
+            if p[2]:
```

Comment:
Using `[2]` here is a bit obscure. We might want to use a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) for registered pipelines if we're going to interact with individual fields.